### PR TITLE
New board page

### LINKED ISSUE #84: Fullstack: CreateBoardPage
## User story

**Who:** **As a** User 
**What:** **I want to**  able to create a new Board using the CreateBoard Page and naturally become the Owner of that same Board
**Why:** **so that I can**  ask others to help me with a decision I want to take.

## Acceptance criteria

The page should look like the one below:
Screenshot 2021-07-30 at 18 31 57

#### -> React Containers and Components 

- [x] Dummy Page - https://github.com/HackYourFuture-CPH/simply-name.it/issues/44
    - [ ] CreateBoardPage - src/client/containers/CreateBoardPage/CreateBoardPage.Container.js
    - [ ] CreateBoardPage - src/client/containers/CreateBoardPage/CreateBoardPage.Style.css
- [x] Arrow Button - https://github.com/HackYourFuture-CPH/simply-name.it/issues/4 (it should link to ProfilePage)
- [ ] Title
- [x] Input - https://github.com/HackYourFuture-CPH/simply-name.it/issues/9 
- [ ] DatePicker for Datetime using the native browser implementation - check here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local (For now ignore the calendar button in purple)
- [ ] Dropzone component - https://github.com/HackYourFuture-CPH/simply-name.it/issues/83
- [x] Buttons for Add members, Reset and Create - https://github.com/HackYourFuture-CPH/simply-name.it/issues/2 (probably needs some adaptation)

#### -> Endpoints
- [x] Endpoint to add a new board - https://github.com/HackYourFuture-CPH/simply-name.it/issues/67

### PR DESCRIPTION
**Pull request by dinythomas89**
# Description

Creating a page to add a new board for a specific user

Fixes #84

# How to test?

npm run dev 
localhost/3000/boards/new-board

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [x] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [x] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by senner007 — CHANGES_REQUESTED** [review]
A few initial comments without looking at the code:
I can see its creating a board, but I get no feedback once the board is created.
Also, I don't see the members for the board in the db?

**Comment by dinythomas89** [comment]
Hi Niels, now the members can be added
But I am little confused how to implement the feedback when board is created. Did you mean an alert message?

**Comment by senner007** [comment]
> Hi Niels, now the members can be added
> But I am little confused how to implement the feedback when board is created. Did you mean an alert message?

Great 👍
I think its better with a message saying that the board is added. Old school alerts are not used anymore

**Comment by dinythomas89** [comment]
> > Hi Niels, now the members can be added
> > But I am little confused how to implement the feedback when board is created. Did you mean an alert message?
> 
> Great 👍
> I think its better with a message saying that the board is added. Old school alerts are not used anymore

Hi Niels, I tried to display a message in below all the components when the board is created and its functioning now

File: src/client/containers/CreateBoardPage/CreateBoardPage.styles.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
+.board-container {
```
Comment (senner007) [inline src/client/containers/CreateBoardPage/CreateBoardPage.styles.css]: Change the css rules please
add 'new-board-' prefix to each

**Review by senner007 — APPROVED** [review]
Great 👍! 
Looks nice and seems to get the job done
You need to throw an ApiError also. See the other prs. Now or create an issue for it
Check the comment about the css classes and merge

**Pull request merged**